web/impact/impact/v1/helpers/profile_helper.py

```diff
@@ -415,14 +415,13 @@ def functional_expertise(self):
 
     @property
     def expert_category(self):
-        return self.field_name("expert_category")
+        return self.get_field_value("expert_category").name
```

Comment:
This will fail if the getattr in the function returns None.  I would expect something like `self.field_property("expert_category", "name")`

Reply:
@mc-nathan I tried something like this first, but I got an error that I was passing in too many values to the `getattr`...  Gonna read up on it now.